pkgs/tools/misc/dgoss/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  # packaging dependencies
+  lib,
+  fetchFromGitHub,
+  resholvePackage,
+  substituteAll,
+  # runtime dependencies
+  bash,
+  coreutils,
+  goss,
+  which,
+}:
+
+resholvePackage rec {
+  pname = "dgoss";
+  version = "0.3.16";
+
+  src = fetchFromGitHub {
+      owner = "aelsabbahy";
+      repo = "goss";
+      rev = "v${version}";
+      sha256 = "1m5w5vwmc9knvaihk61848rlq7qgdyylzpcwi64z84rkw8qdnj6p";
+  };
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p "$out"/bin
+    install extras/dgoss/dgoss "$out"/bin/dgoss
```

Comment:
```suggestion
    install -d extras/dgoss/dgoss $out/bin
```

Reply:
That produced an error: "extras/dgoss/dgoss already exists". I tried with `-D` option